app/messages/components/user-sider.tsx

```diff
@@ -0,0 +1,69 @@
+import AvatarList, { Avatar, AvatarType } from "app/components/avatar-list"
+import { useSiderContext } from "utils/contexts/sider-context"
+
+export type UserSiderProps = React.ComponentProps<typeof AvatarList>
+
+const ScrollbarStyles = () => (
+  <div>
+    <style jsx global>{`
+      ::-webkit-scrollbar {
+        background-color: black;
+      }
+
+      ::-webkit-scrollbar-thumb {
+        background-color: rgba(255, 255, 255, 0.5);
+      }
+    `}</style>
+  </div>
+)
```

Comment:
This may seem akward @cherta but it has an explanation. The thing is that I needed to apply styles to a pseudo-element depending on the `isSiderOpen` prop, but it seems that you cannot render conditional `<style/> tags like this:
```
{ isSiderOpen && (
  <style jsx global>blablabla</style>
)}
```
So I just came up with this, I suspect there might be a better solution, but I'm not sure to be honest.